src/SelfService/Infrastructure/Api/Capabilities/CapabilityController.cs

```diff
@@ -1065,4 +1065,25 @@ public async Task<IActionResult> CreateInvitations([FromRoute] string id, [FromB
             )
         );
     }
+
+    [HttpGet("{id}/configurationlevel")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest, "application/problem+json")]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status401Unauthorized, "application/problem+json")]
+    public async Task<IActionResult> GetConfigurationLevel([FromRoute] string id)
+    {
+        if (!User.TryGetUserId(out var userId))
+        {
+            return Unauthorized();
+        }
+
+        if (!CapabilityId.TryParse(id, out var capabilityId))
+        {
+            return BadRequest(
+                new ProblemDetails { Title = "Invalid CapabilityId provided", Status = StatusCodes.Status400BadRequest }
+            );
+        }
```

Comment:
Should we verify that the user is a member of the capability or is a cloudengineer?
I think that would be a nice touch.